src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/google_tag_manager/head-open.html.twig

```diff
@@ -1,6 +1,4 @@
 <!-- Google Tag Manager -->
-<noscript><iframe src="//www.googletagmanager.com/ns.html?id={{ analytics.content }}"
-height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
 <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
 new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
```

Comment:
Nice job! :+1: Is it possible to set the tag manager before closing the head? Or what is your opion about it?

Reply:
![image](https://user-images.githubusercontent.com/16644200/32542729-dbd21a20-c473-11e7-8ba3-4257bb6da11e.png)
It should be as close as possible to the open tag? Or is this information wrong?
https://developers.google.com/tag-manager/quickstart

Reply:
@trickreich good point! :+1: